controllers/follow.js

```diff
@@ -0,0 +1,82 @@
+
+/**
+ * Controller for follow and unfollow
+ */
+export default class FollowController {
+  /**
+   * @description follow a user
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns  {object} response
+   */
+  static async followUser(req, res) {
+    const { userFollowing, userToFollow, userToFollowId } = req;
+    // Check if user is already following the user and return an error if they are
+    const getFollowing1 = await userFollowing.getFollowing();
```

Comment:
@youngestdj why is this `getFollowing1`? Is there `getFollowing2`?

Reply:
This has been removed